mlflow/pyspark/optuna/study.py

```diff
@@ -0,0 +1,188 @@
+import datetime
+import logging
+import sys
+from collections.abc import Callable, Iterable
+from typing import Optional
+
+try:
+    import optuna
+    from optuna import pruners, samplers, storages
+    from optuna.study import Study
+    from optuna.study._optimize import _optimize_sequential
+    from optuna.trial import FrozenTrial
+except ImportError:
+    sys.exit()
```

Comment:
what is the purpose of this line?

Reply:
we should avoid using sys.exit(), but raise exception instead.